pandas/tests/indexing/test_check_indexer.py

```diff
@@ -26,8 +26,8 @@
     ],
 )
 def test_valid_input(indexer, expected):
-    array = np.array([1, 2, 3])
-    result = check_array_indexer(array, indexer)
+    n_arr = np.array([1, 2, 3])
```

Comment:
this looks like it could mean `number of arrays`. `arr` will work here, I just tried it